couchbase/management/bucket_settings.hxx

```diff
@@ -99,17 +99,17 @@ struct bucket_settings {
     std::string name;
     cluster::bucket_type bucket_type{ cluster::bucket_type::unknown };
     std::uint64_t ram_quota_mb{ 100 };
```

Comment:
I think we want this to be 0 MB, right?  For create I don't think it matters, but for update_bucket this might be a logic error if the user does not intend to update the buckets RAM quota.

Reply:
Yep, I forgot to update it - fixed now